ext/mro/mro.xs

```diff
@@ -299,6 +322,41 @@ __dopoptosub_at(const PERL_CONTEXT *cxstk, I32 startingblock) {
 
 MODULE = mro		PACKAGE = mro		PREFIX = mro_
 
+#ifdef PERL_IMPLICIT_CONTEXT
+
+void CLONE (...)
+CODE:
+#undef memcpy
+#define memcpy(a,b,c) NOOP
+    {
+        MY_CXT_CLONE; /* possible declaration */
+        init_MY_CXT(aTHX_ aMY_CXT);
+    }
+#undef memcpy
```

Comment:
Did you consider fixing the API here (eg. adding `MY_CXT_NEW`) instead of this hack on memcpy()?

Reply:
Out of scope until https://github.com/Perl/perl5/pull/22879 is closed/resolved.